samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -134,8 +136,9 @@ static void radio_config(nrf_radio_mode_t mode, enum transmit_pattern pattern)
 {
 	nrf_radio_packet_conf_t packet_conf;
 
-	/* Reset Radio ramp-up time. */
-	nrf_radio_modecnf0_set(NRF_RADIO, false, RADIO_MODECNF0_DTX_Center);
+	/* Set fast ramp-up time. */
+	nrf_radio_modecnf0_set(NRF_RADIO, true, RADIO_MODECNF0_DTX_Center);
```

Comment:
From my perspective it looks good but it would be good to know why fast ramp-up wasn't used from the beginning for few. But for me it is ok to go with it